src/client/src/App.test.tsx

```diff
@@ -1,9 +1,18 @@
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import { shallow } from "enzyme";
+
 import App from "./App";
 
-it("renders without crashing", () => {
-  const div = document.createElement("div");
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+describe("App", () => {
```

Comment:
You might want to be more descriptive in the describe statement. I see that for now, we are not testing it a lot but I think a description like "App Component Renders" would be more intuitive.

Reply:
that's done in the `it` statements

Reply:
I agree, your `it` statement says that `renders without crashing`.
There can be more `it` statements within the `describe` statement that say for instance: `renders a button` or `crashes for a particular prop`. I feel we should have a more descriptive `describe` statement. This would really help as we would be developing more tests.

Reply:
Okay sure, different examples did it differently but i'm open to this

Reply:
IMO it's ok. I usually see describe("Tested component", ...)

Reply:
Sure :)